starter-code/database.js

```diff
@@ -144,16 +129,18 @@ class Database {
       if (error) {
         callback(error)
       } else {
-        // LAB 8
-        // Implement the query to review a product
-        // productName is the name of the product to review
-        // review is the document to insert
-        // remeber once it's finish to comment callback('Error reviewing product');
-        
-        callback('Error reviewing product');
+        database.collection('products').updateOne({"products":productName},
```

Comment:
This is not working, can you guess why?

Tip: you don't want to search the "products" that match productName.